backend/src/main/java/de/webdev/backend/security/AppuserRole.java

```diff
@@ -2,5 +2,7 @@
 
 public enum AppuserRole {
```

Comment:
not sure how you have it in the rest of your code, but maybe AppUser instead of Appuser?
```suggestion
public enum AppUserRole {
```